recipe/meta.yaml

```
@@ -13,7 +13,7 @@ source:
     folder: gh
 
 build:
-  number: 0
+  number: 1
```

```suggestion
  number: 0
```
0 still not released